conf/gulp-tasks/bundle/bundle.js

```diff
@@ -88,7 +94,13 @@ exports.legacyBundle = function (callback, outputConfig, bundleName, locale, lib
           '@babel/plugin-transform-object-assign'
         ]
       })
-    ]
+    ],
+    onwarn: function(warning) {
```

Comment:
I think I would rather use the commonjs bundle than have this suppression, if the bundle size is basically the same